action.yml

```diff
@@ -29,6 +29,10 @@ inputs:
   project-url:
     description: 'Project URL'
     required: false
+  retry-attempts:
+    description: 'The number of retry attempts to download Testcontainers Cloud Client before giving up'
+    required: false
+    default: '0'
```

Comment:
can be 1 by default

```suggestion
    default: '1'
```

Reply:
And that would be different from the existing behaviour without retries

Reply:
I think it's a sane default that adds reliability, despite it's different by default